data/human/human missions.txt

```diff
@@ -3601,3 +3601,84 @@ mission "Mafia Extortion"
 			label payup
 			`	The man takes your credits, responds with a brief "Thank you," and walks away.`
 				accept
+
+
+
+mission "Repulsor failure"
+	landing
+	# Don't require the player to pay credits again if the player was already allowed to land when this mission triggered.
+	clearance
+	to offer
```

Comment:
```suggestion
	clearance
	source
		not attributes "station"
		not attributes "gas giant"
		not attributes "uninhabited"
		not attributes "korath"
	to offer
```

Worth considering whether or not it should also be restricted from triggering on alien worlds. I think it's fine if it triggers on Remnant worlds, but given the player can't even speak Korath, a casual fining by the authorities doesn't seem to fit... It's not a *perfect* fit for the Remnant, but I think it's close enough to be fine. As far as that goes, I think it's *probably* fine on any alien world where the player can speak the language. It's kind of incongruous when they can't. Simple solution is just not allow it on said alien worlds. (ex. Wanderer, coalition) more complex solution is tie it to the translators so this can trigger once the player can speak to the locals, but it won't happen anywhere they *can't* speak to them.